packages/botonic-plugin-flow-builder/src/handoff.ts

```diff
@@ -36,15 +42,19 @@ export async function doHandoff(
   // }
 
   await handOffBuilder.handOff()
+
+  if (flowBuilderPlugin.trackEvent) {
+    await flowBuilderPlugin.trackEvent(request, 'HANDOFF_SUCCESSFULL', {
+      queueName: queueFound?.name,
```

Comment:
i think this event should be more similar to the `QUEUE_STATUS` event.
One only tracks the `queueName` and the other the `queue_id` and `queue_name`. Also one is in camel case and the other in snake case